src/main/java/data/CameraTimeline.java

```diff
@@ -12,6 +14,9 @@
     // The camera that is associated with this timeline.
     @Getter @Setter
     private Camera camera;
+    
+    @Getter
+    private ArrayList<CameraShot> shots;
```

Comment:
The DirectorTimeline already has a list of DirectorShots, which is why I did the same for CameraTimeline. I could either remove both of these and use the list from TimeLine, or leave it as it is now